bin/utils/update_browsers.js

```diff
@@ -0,0 +1,75 @@
+/* global __dirname */
```

Comment:
setting the environment to node in the eslintrc doesn't imply this?

Reply:
Right.